R/req-perform-stream.R

```diff
@@ -53,19 +53,30 @@ req_perform_stream <- function(req,
     }
 
     if (length(buf) > 0) {
+      # there are leftover bytes, but the stream is complete
+      # break the loop so that the callback() is given the
+      # whole buffer
+      if (!incomplete) {
```

Comment:
Work it work to move this higher?  i.e. make it the other side of the `if (incomplete)` statement above.